### PR TITLE
bug(authentication): remove token verification when signing in 

### PR DESCRIPTION
**Pull request by Boths1229**
#### What does this PR do?
- Implement my user sign in token
- Remove token as a requirement for sign in

#### Description of the task to be completed?
- Remove token as part of requirement for user sign in. 
- Update sign-in tests for conformity
- Worked on my  sign in functions in my controller users.js, removed verify token
- Also updated my gh-pages 

#### How can this manually be tested?
- clone this repo
```
$ git clone https://github.com/Boths1229/quick-credit.git
```
- install all dependencies
```
$ npm i
```
- start Application
```
$ npm start:dev
```
- log on to _localhost:<PORT>_ to interact with endpoint created

#### Relevant PT story(ies)
#166008700[https://www.pivotaltracker.com/n/projects/2326722]

[finishes #166008700]

### CODE REVIEW

File: server/test/test.js [hunk 1]
```diff
@@ -219,21 +219,15 @@ describe('POST should return you are not up to age to apply', () => {
 });
 
 describe('POST api/v1/auth/signin', () => {
-  const token = jwt.sign({
-    email: 'boths104@example.com',
-    firstName: 'sonmajkkl',
-    lastName: 'Enyiokwakll',
-  },
-  'secret');
+  const token = 'ajdsdkdk.akskks29030303umds.s';
```
Comment (damafeez) [inline server/test/test.js]: You should remove this variable, YOU DON'T NEED A TOKEN WHEN SIGNING IN

File: server/test/test.js [hunk 2]
```diff
@@ -219,21 +219,15 @@ describe('POST should return you are not up to age to apply', () => {
...
   it('should return signin successful', (done) => {
     chai.request(server)
       .post('/api/v1/auth/signin')
       .set('authorization', token)
```
Comment (damafeez) [inline server/test/test.js]: You don't need to set authorisation token in the header while signing in.

File: server/test/test.js [hunk 3]
```diff
@@ -219,21 +219,15 @@ describe('POST should return you are not up to age to apply', () => {
...
       .set('Accept', 'application/json')
       .send({
         email: 'boths104@example.com',
-        password: 'developer',
-        token,
+        password: 'developer'
       })
       .end((err, res) => {
         expect(res.body).to.be.an('object');
```
Comment (damafeez) [inline server/test/test.js:230]: You should check also that the returned object contains a token.

**Review by damafeez — CHANGES_REQUESTED** [review]
Hi @Boths1229, you've done an awesome job altogether.
However, I don't think you commit message is detailed enough:
- the commit body should contain a breakdown (in steps) of how you have achieved the functionality in the commit header.

I think a commit message like this might be more appropriate
```
bug(authentication): remove token verification when signing in

- remove token requirement on sign-in
- update sign-in tests for conformity
- another thing I did
- yet another thing I did e.t.c

[finishes #166008700]
```
Also, your PR's title should not contain the footer `[finishes #166008700]`

Please attend/respond to the feedback

File: server/test/test.js (hunk 2 above)
Comment (Boths1229) [reply server/test/test.js]: i did that.

Comment (damafeez) [reply server/test/test.js]: You did?
Why are you passing token via the header in this test?

**Review by damafeez — CHANGES_REQUESTED** [review]
Great job implementing the suggested changes.
Please ensure your commits follow the recommended convention then you can merge.

You can modify your commit messages by doing `git commit --amend`

**Pull request merged**